packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/index.scss

```diff
@@ -120,7 +120,7 @@
             right: 1.6rem;
             bottom: 1.6rem;
             border-radius: 3rem;
-            background-color: $color-grey-2;
+            background-color: var(--general-section-6);
```

Comment:
after line 68 under &__title, could you add "color:var(--text-general)" as the title text is not visible on dark theme

<img width="589" alt="Screenshot 2022-11-29 at 12 08 44 PM" src="https://user-images.githubusercontent.com/102643568/204436545-1f954389-ecc7-4a9f-a975-3630c0f4c271.png">

Reply:
@farabi-deriv  line 68 has been changed, background-color I left because the comment sounded like this 'Zoom in/out button is not visible on preview section under the dashboard tab', I think it is about sign '+' and '-'. They are not visible..

Reply:
@maryiafrantsava-binary no need to change the background-color here. I only meant to add the "color:var(--text-general)" after line 68 in this file. Thanks for updating.